js/apps/system/_admin/aardvark/APP/frontend/js/collections/arangoQueries.js

```diff
@@ -13,7 +13,7 @@
     initialize: function (models, options) {
       var self = this;
 
-      $.ajax('whoAmI?_=' + Date.now(), {async: true}).done(
+      $.ajax('whoAmI?_=' + encodeURIComponent(Date.now()), {async: true}).done(
```

Comment:
`Date.now()` returns a number. I don't think this change does anything meaningful.